src/PowerShell.Core.Instrumentation/README.md

```diff
@@ -0,0 +1,5 @@
+# PowerShell.Core.Instrumentation
+
+The code under `PowerShell.Core.Instrumentation` is being migrated to [PowerShell-native](https://github.com/PowerShell/PowerShell-native) repository.
+Please make PRs to the new repository.
+Code under here will be removed once the move is complete.
```

Comment:
I'd be happy to see a guidelines about new workflow for dev/debug and release. (Scenario - "I need use new Unix syscall")